fw/http_tbl.h

```diff
@@ -56,5 +56,6 @@ typedef struct {
 
 TfwVhost *tfw_http_tbl_vhost(TfwMsg *msg, bool *block);
 int tfw_http_tbl_method(const char *arg, tfw_http_meth_t *method);
+const char *tfw_http_tbl_method_get_name(int value);
```

Comment:
This is different subsystem https://github.com/tempesta-tech/tempesta/wiki/HTTP-tables , which has nothing with cache, so we shouldn't use any routines from it in this patch.

Reply:
So I should double the function of getting string from method MACRO in this module?
The only purpose of this call is to get the "purge" string from TFW_HTTP_METHOD_PURGE. Or I can fix the input method to PURGE only and avoid any transformations.

Reply:
No, I commented at the place of using the function: just hardcode "PURGE" as you already hardcoded "GET"

Reply:
ok, done